README.md

```diff
@@ -1,32 +1,67 @@
+# LoTAS
+Used to create non-playback, Low optimization Tool-Assisted Speedruns (LoTAS).  
+Essentially, you record the game in slow motion, record yourself speedrunning with a recording program such as [OBS](https://obsproject.com),  
+then speed up the video footage.  
 
-## How do I start?
+LoTAS is a collection of tools commonly used for TASing, with a big focus on RNG manipulation.  
 
-Once you installed the Mod and opened a World you can access the TAS Features by pressing `Escape`.
-This might look confusing to you, but all you need to know is that the top left buttons + and -, increase or decrease the speed of the game.
-Have a look in the [Wiki](https://github.com/MCPfannkuchenYT/LoTAS/wiki) to see the rest of the features explained
+## Tools
+- *Tickratechanger*: Used for slowing down the game speed
+- *Savestates*: Used for creating backups of your world, basically quick save/quick load
+- *Drop Manipulation*: Manipulates mob drops to always drop the best items
+- *Dragon Manipulation*: Manipulates the dragon phases
+- *Item Duplication (Dupemod)*: Replicates the duping bug from vanilla, without the need to close the game.
+- *Spawn Manipulation*: Spawn entities at a certain location.
+- *AI Manipulation*: Let entities pathfind to a certain place  
 
+*and more...*
 ## Bugs / Feature Requests
+We won't accept any feature-requests at this time, as well as bug reports.
 
-Report Bugs under the Issues Section or on our [Discord](https://discord.gg/minecraft-tas-373166430478401555)
-All Feature Requests go also onto the Discord Server.
+If you need help with setting up LoTAS then join this [Discord](https://discord.gg/minecraft-tas-373166430478401555)
 
 ## Translations
-You can find the file under [src/main/resources/assets/lotas/lang/en_us.lang](https://github.com/MinecraftTAS/LoTAS/blob/dev/LoTAS-Fabric/src/main/resources/assets/lotas/lang/en_us.lang)
-
 Credits:  
-Simplified Chinese: [RayXu6](https://github.com/Naruyoko), Proofreading by NormalZombie#2431  
+Simplified Chinese: [RayXu6](https://github.com/Naruyoko), Proofreading by normalzombie  
 Japanese: [Naruyoko](https://github.com/Naruyoko)  
 Korean: Sidite  
 French: azulamazigh  
 
-## Features
-* Duping: Save your Inventory/Items/Chests with 1 button and load' em again with another click!
-* Savestates: Create copies of your World and load them again without copying the world manually!
-* Drop Manipulation: Need Flint? Get Flint!
-* Spawn Manipulation: Spawn Mobs everywhere possible using the Gui!
-* AI Manipulation: Change what the Dragon should do and where your Entities should move.
-* And More: There are more features like: Tick Indicator, Timer, No Damage and more.. all of those in the wiki.
+# FAQ
+## How do I start?
+- Most functionality is in the in-game pause menu via buttons.
+- The main menu has a "configuration" button
+- Pressing F6 opens an in game HUD, where you can enable information or an in-game timer.
```

Comment:
- Most of LoTAS is accessible via the in-game pause menu.
[...]
- Pressing F6 [...], where you can display various informations onscreen.